testing/e2e/src/routes/byok.tsx

```
@@ -0,0 +1,95 @@
+import { createFileRoute } from '@tanstack/react-router'
+import { useMemo, useRef } from 'react'
+import { fetchServerSentEvents, useChat } from '@tanstack/ai-react'
+import {
+  ByokKeyManager,
+  ByokProvider,
+  byokHeaders,
+  memoryStorage,
+  useByok,
+} from '@tanstack/ai-byok/react'
+import { ChatUI } from '@/components/ChatUI'
+import type { Keyring, KeyringStorage } from '@tanstack/ai-byok/react'
```

ESLint

[error] `@tanstack/ai-byok/react` type import should occur before import of `@/components/ChatUI`

(import/order)